### DISCUSSION TITLE
Add parallel translator selector bar to the pop-up

### DISCUSSION THREAD

**Discussion by czarnyckm** [you-apps/TranslateYou#464] [Ideas]
If the translate is triggered from context menu of marked text the pop-up is shown. Sadly it uses the primary selected translation provider only.
It will be good to have possibility to check different translator results and to have an option to show the selected text tranaslation directly in the app instead of the pop-up.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by Bnyro** [comment]
The dialog has a limited amount of space since it doesn't fill the full screen, thus it's meant to only show the necessary information for a basic translation.

Therefore I don't think that's a good idea, given the lack of free space.

**Comment by asereze** [comment]
Couldn't a button be added, perhaps next to the title (as in the example image I made), that would open a list of translation providers to select the one you want, like what happens with the languages? That would save up space.

![photo_2024-11-04_20-40-12](https://github.com/user-attachments/assets/fa41706f-c9e1-45d7-a1d0-9a7592c3650f)

**Reply by czarnyckm** [reply]
On click it will be  toggle all enabled translators and optionally on long click show translator selection pull down menu.
It will be good to quick check out results.

**Reply by Bnyro** [reply]
I agree, good idea @asereze :+1:

**Comment by Bnyro** [comment]
https://github.com/you-apps/TranslateYou/commit/dd782c4d25e625695f53539c41084da2dbe8a288

**Reply by asereze** [reply]
Thank you!
